settings.gradle

```diff
@@ -1 +1,3 @@
-include ':AndroidStealth', ':libraries:FileChooser', ':libraries:droidmorphing-lib', ':libraries:DirectoryChooser'
+include ':AndroidStealth', ':libraries:FileChooser', ':libraries:DirectoryChooser', ':droidmorphing-lib'
+project(':droidmorphing-lib').projectDir = new File('libraries/droid-morph/droidmorphing-lib')
```

Comment:
As far as I understand gradle, it tells it to just include `:droidmorphing-lib` project as a dependency which can be found at the defined path, without doing any magic which gradle would normally do, thus allowing us to compile it as usual in `build.gradle`.